vignettes/miaSim.Rmd

```diff
@@ -137,12 +145,103 @@ crmodel <- simulateConsumerResource(n_species = 2,
              n_resources = 4,
 	     E = randomE(n_species = 2, n_resources = 4))
 
+miaViz::plotSeries(crmodel, x = "time")
+
+# example to get relative abundance and relative proportion of resources
+#'norm = TRUE' can be added as a parameter.
+
+# convert to relative abundance
+ExampleCR <- mia::transformCounts(crmodel, method = "relabundance")
+
+miaViz::plotSeries(ExampleCR, x = "time")
+```
+
+
+```{r crmodel, eval=FALSE}
+#Recommended standard way to generate a set of n simulations (n=2 here) from a given model
+n_simulations <- lapply(seq_len(2), function (i) {do.call(simulateConsumerResource, params)})
```

Comment:
how about just "simulations" because "n_simulations" looks like a number (n) but this is not a number.